README.md

```diff
@@ -42,8 +34,15 @@ Create a `config.yaml` for your migration using the template `config.yaml.exampl
 
 The Scylla Migrator is built against Spark 3.5.1, so you'll need to run that version on your cluster.
 
-If you didn't build Scylla Migrator on the master node:
-After running `build.sh`, copy the jar from `./migrator/target/scala-2.13/scylla-migrator-assembly-0.0.1.jar` and the `config.yaml` you've created to the Spark master server.
+Download the latest [release](https://github.com/scylladb/scylla-migrator/releases) of the migrator:
+
+~~~ sh
+wget https://github.com/scylladb/scylla-migrator/releases/latest/download/scylla-migrator-assembly.jar
```

Comment:
This won’t work until we publish a first release, but you can see how it looks like from my fork:

https://github.com/julienrf/scylla-migrator/releases